rome_app/views.py

```diff
@@ -497,6 +505,7 @@ def essay_detail(request, essay_slug):
     context=std_context(request.path, style="rome/css/essays.css")
     context['essay_text'] = essay.text
     context['essay'] = essay
+    context['author'] = essay.author
```

Comment:
You already have the essay object in the context. Could you take out this separate "author" context item, and then in the essay_detail template above, you can just use "essay.author" instead of "author".